packages/sdk-core/src/BacktraceCoreClient.ts

```diff
@@ -178,6 +178,13 @@ export abstract class BacktraceCoreClient {
         this._enabled = true;
     }
 
+    /**
+     * Uses client as the global client.
+     */
+    public static use(client: BacktraceCoreClient) {
```

Comment:
We should control that, we shouldn't expose this option outside. If someone uses it and re-initialize client, we might have a huge issue with our native integrations and also with our callback functions (like breadcrumbs etc). The Backtrace client instance is protected. Each client can use it. Why do we need this?

Reply:
Removed the method and changed how the module is initialized.